app/javascript/packages/address-search/webpack.config.cjs

```diff
@@ -17,6 +17,7 @@ module.exports = /** @type {import('webpack').Configuration} */ ({
   },
   resolve: {
     extensions: ['.js', '.jsx', '.ts', '.tsx', '.mjs', '.cjs', '.mts', '.cts'],
+    conditionNames: ['source'],
   },
   externals: /^(?!(@18f\/identity-|\.))/,
```

Comment:
since this is changing webpack config, maybe warrants a minor version update?

Reply:
We haven't published 2.3.0 or 2.4.0 so I think bumping the version is unnecessary. Then we're just skipping more versions. If we had already published 2.4.0 I'd think about it more (but even then I lean towards no, as this doesn't change the package's interface)